api/argo-templates/ncn/worker.wipe-and-reboot.yaml

```diff
@@ -33,5 +33,9 @@ tasks:
           value: "{{ `{{inputs.parameters.dryRun}}` }}"
         - name: scriptContent
           value: |
-            TODO
+            mv /etc/cray/upgrade/csm/{{ `{{inputs.parameters.targetNcn}}` }}/state \
+                /etc/cray/upgrade/csm/{{ `{{inputs.parameters.targetNcn}}` }}/state.bak || true
```

Comment:
Use *install* instead of *mv || true*, mv isn't idempotent and forcing the true means we could ignore errors that might be useful (read only filesystem etc...)

Reply:
Either that or [[ -f ... ]] && mv would work as well now that I think of it if the || true is to handle if the source is missing.